SSDynamicText/NSAttributedString+SSTextSize.h

```diff
@@ -3,10 +3,10 @@
 //  SSDynamicText
 //
 //  Created by Remigiusz Herba on 28/08/15.
-//
+//  Copyright (c) 2015 Splinesoft. All rights reserved. 
 //
 
-#import <Foundation/Foundation.h>
+@import Foundation;
```

Comment:
Look like cocoapods changed this flag to enabled by default. But I added to be proof for future changes.